src/errors.js

```diff
@@ -16,21 +16,6 @@ class NonReversibleMigrationError extends Error {
 NonReversibleMigrationError.code = 'ERR_NON_REVERSIBLE_MIGRATION'
 exports.NonReversibleMigrationError = NonReversibleMigrationError
 
-/**
- * Exception raised when structure of a repo is not as expected.
- */
-class UnknownRepoStructureError extends Error {
-  constructor (message) {
-    super(message)
-    this.name = 'UnknownRepoStructureError'
-    this.code = 'ERR_UNKNOWN_REPO_STRUCTURE'
-    this.message = message
-  }
-}
-
-UnknownRepoStructureError.code = 'ERR_UNKNOWN_REPO_STRUCTURE'
-exports.UnknownRepoStructureError = UnknownRepoStructureError
```

Comment:
Because I have moved the check if a repo is initialized to [`getVersion`](https://github.com/ipfs/js-ipfs-repo-migrations/pull/3/files#diff-9314bebf539a650be3efc61e30b60697R22) and because of this change, this error would never have been called as `NotInitializedRepo` error will be thrown first.